package.json

```diff
@@ -41,5 +41,8 @@
   "lint-staged": {
     "*.{ts,tsx,js,json,css,md}": "prettier --write"
   },
-  "prettier": {}
+  "prettier": {},
+  "resolutions": {
+    "esbuild": "0.15.13"
+  }
```

Comment:
`esbuild@0.15.8` is needed as it comes with bundling fixes: https://github.com/evanw/esbuild/releases/tag/v0.15.8